dnd-troops-web/src/main/resources/Texts.properties

```diff
@@ -51,6 +51,32 @@ hero.search.any_role=Any role
 hero.search.any_race=Any race
 hero.search.role=Role
 
+#group
+group.all.title=Groups
+group.all.add=Add group
+group.name=Name
+group.heroes=Heroes
+group.heroes.role=Roles
+group.heroes.add=Add heroes
+group.heroes.remove=Remove heroes
+group.delete.problem=Problem while deleting group
```

Comment:
Add comment that only groups without heroes can be deleted, so tester know its not bug but feature.